website/docs/docs/core/connect-data-platform/hive-setup.md

```diff
@@ -72,15 +72,15 @@ your_profile_name:
   outputs:
     dev:
```

Comment:
There are more examples higher up that need replacing. Like the [Insecure section](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/hive-setup#insecure). And the example [here](https://docs-getdbt-mdk5plxur-dbt-labs.vercel.app/docs/core/connect-data-platform/hive-setup#connection-methods)